lib/tasks/bundlers/generateComponentPreload.js

```diff
@@ -14,24 +15,32 @@ const ReaderCollectionPrioritized = require("@ui5/fs").ReaderCollectionPrioritiz
  * @param {module:@ui5/builder.tasks.TaskUtil|object} [parameters.taskUtil] TaskUtil
  * @param {object} parameters.options Options
  * @param {string} parameters.options.projectName Project name
- * @param {Array} [parameters.options.paths] Array of paths (or glob patterns) for component files
- * @param {Array} [parameters.options.namespaces] Array of component namespaces
+ * @param {string[]} [parameters.options.excludes=[]] List of modules declared as glob patterns (resource name patterns)
+ * that should be excluded. A pattern either contains of a trailing slash '/' or single '*' and double '**' asterisks
```

Comment:
"...contains **~of~** a trailing slash..." or something like that. Currently, this sounds somewhat strange

Reply:
Updated JSDoc in generateComponentPreload, generateLibraryPreload and moduleBundler (for `filters`) to this:

>  A pattern ending with a slash '/' will, similarly to the use of a single '*' or double '**' asterisks,
>  denote an arbitrary number of characters or folder names.